src/main/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -1564,23 +1553,15 @@ public SAMTagAndValue(final String tag, final Object value) {
         return ret;
     }
 
-    Integer getIndexingBin() {
-        return mIndexingBin;
-    }
-
-    /**
-     * Used internally when writing BAMRecords.
-     * @param mIndexingBin c.f. http://samtools.sourceforge.net/SAM1.pdf
-     */
-    void setIndexingBin(final Integer mIndexingBin) {
-        this.mIndexingBin = mIndexingBin;
-    }
-
     /**
      * Does not change state of this.
```

Comment:
update comment to reflect the fact that there is a possible side-effect due to the need to parse the cigar.

Reply:
Done.